src/cli.zig

```diff
@@ -186,7 +186,7 @@ pub const Arguments = struct {
     const base_params_ = (if (Environment.isDebug) debug_params else [_]ParamType{}) ++ [_]ParamType{
         clap.parseParam("--env-file <STR>...               Load environment variables from the specified file(s)") catch unreachable,
         clap.parseParam("--cwd <STR>                       Absolute path to resolve files & entry points from. This just changes the process' cwd.") catch unreachable,
-        clap.parseParam("-c, --config <PATH>?              Specify path to Bun config file. Default <d>$cwd<r>/bunfig.toml") catch unreachable,
+        clap.parseParam("-c, --config <PATH>               Specify path to Bun config file. Default <d>$cwd<r>/bunfig.toml") catch unreachable,
```

Comment:
This is a big breaking change as people may have `bun -c ./index.ts` in their scripts to use local `bunfig.toml`. However in this PR it already does that by default (as its what most expect now), so this could cause problems...

Reply:
done because technically its not needed, but this appears to create more problems rn, will consider this change in followup (or allowing `-c` itself at very least)